addons/website/models/res_config.py

```diff
@@ -50,3 +77,20 @@ def set_google_maps_api_key(self):
     def get_default_google_maps_api_key(self, fields):
         google_maps_api_key = self.env['ir.config_parameter'].get_param('google_maps_api_key', default='')
         return dict(google_maps_api_key=google_maps_api_key)
+
+    @api.depends('google_analytics_key')
+    def _compute_has_google_analytics(self):
```

Comment:
Do a true multi. I know it is not really necessary as there is always one active record, but it is better to be multi.